src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Reload/ReloadableProject.cs

```diff
@@ -127,9 +131,11 @@ public Task Initialize()
                         var tw = new StringWriter();
                         msbuildProject.Save(tw);
                     }
-                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException)
+                    catch (Microsoft.Build.Exceptions.InvalidProjectFileException ex)
                     {
                         // Indicate we weren't able to complete the action. We want to do a normal reload
+                        var faultEvent = new FaultEvent("vs.projectsystem.projectreload.reload-failed", "Project Reload Failure due to invalid project file", ex);
```

Comment:
ah, it is fault. if you want to post NFW, use this API instead. 

http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/Watson/WatsonReporter.cs,28